wandb/proto/wandb_internal.proto

```diff
@@ -493,9 +493,6 @@ message Request {
     RunStartRequest       run_start = 11;
     CheckVersionRequest   check_version = 12;
     LogArtifactRequest    log_artifact = 13;
-    ArtifactSendRequest   artifact_send = 14;
-    ArtifactPollRequest   artifact_poll = 15;
-    ArtifactDoneRequest   artifact_done = 16;
```

Comment:
if you want to delete these message probably best practice to reserve fields, but given that it was only used with the grpc interface, maybe it is fine?

https://protobuf.dev/programming-guides/dos-donts/#reserve-tag-numbers

Reply:
oh wait, i think it's totally fine to remove these, since these are requests we don't persist these in the transaction log. Ignore my previous message